python/lsst/pipe/tasks/measurePsf.py

```diff
@@ -121,15 +123,12 @@ class MeasurePsfTask(pipeBase.Task):
 
 @section pipe_tasks_measurePsf_Example	A complete example of using MeasurePsfTask
 
-This code is in @link measurePsfTask.py@endlink in the examples directory, and can be run as @em e.g.
+This code is in `measurePsfTask.py` in the examples directory, and can be run as @em e.g.
 @code
 examples/measurePsfTask.py --doDisplay
 @endcode
 @dontinclude measurePsfTask.py
 
-The example also runs SourceDetectionTask and SingleFrameMeasurementTask;
```

Comment:
Isn't this still true?

Reply:
As far as I can tell, the `meas_algorithms_measurement_Example` target hasn't existed in years, and I can't reconstruct what it was originally meant to explain.

Reply:
Restored just in case.